recipes/clazy/all/test_package/conanfile.py

```diff
@@ -0,0 +1,13 @@
+from conan import ConanFile
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "VirtualBuildEnv"
+    test_type = "explicit"
+
+    def build_requirements(self):
+        self.tool_requires(self.tested_reference_str)
+
+    def test(self):
+        self.run("clazy --version")
```

Comment:
This won't work for crossbuild scenarios in the CI, a better approach is to just check for existence of the executable

```suggestion
    def requirements(self):
        self.requires(self.tested_reference_str)

    def test(self):
        assert os.path.exists(os.path.join(self.cpp.build.bindir, "clazy"))
```